让screenlog始终在最上面

guomindong opened the pull request:
为var c增加z-index:10000

guomindong committed:
让screenlog始终在最上面

为var c增加z-index:10000

chinchang commented:
You should modify only `screenlog.js` and not `screenlog.min.js`. Minified file will be generated once this pull request is merged inm aster

guomindong committed:
让screenlog所属的div始终在最上面

var div = createElement( 'div', 'z-index:9999,font-family:Helvetica,Arial,sans-serif;font-size:10px;font-weight:bold;padding:5px;text-align:left;opacity:0.8;position:fixed;right:0;top:0;min-width:200px;max-height:50vh;overflow:auto;background:' + options.bgColor + ';color:' + options.color + ';' + options.css);

chinchang commented:
fixed separately in master. live in 0.2.2
